packages/components/src/sandbox/index.js

```diff
@@ -137,8 +137,12 @@ class Sandbox extends Component {
 			body {
 				margin: 0;
 			}
+			html,
+			body,
+			body > div,
 			body > div > iframe {
 				width: 100%;
+				height: 100%;
```

Comment:
Note that this height property is necessary in order for the responsiveness to work, basically because every element that contains the embed needs an explicit height in order for a single percentage height to work properly. 

But if this component is used for things that aren't aspect ratio responsive, we need to scope it. Something like:

```
html.is-video,
html.is-video body,
html.is-video body > div,
html.is-video body > div > iframe {
height: 100%;
}
```

Reply:
Ah, I see :) Ok, I'll see how it can be changed around to only get applied to things we're doing aspect ratio preservation on.

Reply:
Feel free to send this PR right back at me if we can have a CSS class or separate style declaration for aspect ratio responsive stuff only.

Reply:
It would be cool to have an `.is-responsive` or `.has-aspect-ratio` class to target here.

Reply:
Agreed. I was thinking `.wp-embed-has-aspect-ratio` and then the editor should be able to use that to apply the height to the sandbox iframe.